docs/reqs/reqs.tex

```diff
@@ -211,8 +218,9 @@ \subsection{Non-RPAS Air-Ground I/O}
 
 \subsection{Computing Platform}
 \begin{enumerate}[{F.CP}.1:]
-	\item Captures video at a minimum resolution of 640x480 at 10fps.
-	\item Onboard FPGA is capable of performing parallel matrix computation to speedup machine learning.
```

Comment:
These are non function requirements. The high level requirements would fit well here though (I meant to select line 221-223)